### ISSUE TITLE
Add simple mode "find duplicate filenames"

### ISSUE THREAD

**Issue by svenb1234** [andrey18106/mediadc#46]
**Description**
The easiest duplicates are those that have the same filename and filesize. However currently there is no way to restrict the search to only find these kind of duplicates.

**Example**
Due to several backups and copies, duplicate files/pictures and videos are scattered all over the system. In order to find those duplicates it would be helpful to restrict the search to identical files.

**Comment by ksihota** [comment]
Yes, this is one of the simplest checks but would take care of a huge number of duplicate files that are simply copies in different directories. The only issue I can see is how the duplicates are displayed for deleting. Currently I have to hover over to see the actual path and then I have to delete each one manually which requires 2 clicks because of the popup verification. Are these things customizable in settings somewhere? I would prefer some way to select a number of images and delete them with a single command. If they were displayed in a grid by location it would be easy to tell where they are and it could also provide the option of selecting all duplicates in a specific location.
